fbu-app/ViewControllers/HomeScreens/AddHousemateViewController.m

```diff
@@ -19,6 +19,9 @@ - (IBAction)tapAddHousemates:(id)sender;
 @property (nonatomic, strong) NSMutableArray *potentialHousemates;
 @property (nonatomic, strong) NSMutableArray *housematesToAdd;
 @property (weak, nonatomic) IBOutlet UIButton *houseButton;
+@property (strong, nonatomic) Persona *currentPersona;
```

Comment:
if properties are being used locally only they can be made into instance variables instead
Ex:

    @interface Class ()
    {
        NSString string;
        NSUInteger number;
    }

and then you dont have to access them by using self